examples/tutorials/curation/plot_1_automated_curation.py

```diff
@@ -0,0 +1,248 @@
+# and only manually curate those which it is less sure of.
+
+
+def calculate_moving_avg(label_df, confidence_label, window_size):
+
+    label_df[f'{confidence_label}_decile'] = pd.cut(label_df[confidence_label], 10, labels=False, duplicates='drop')
+    # Group by decile and calculate the proportion of correct labels (agreement)
+    p_label_grouped = label_df.groupby(f'{confidence_label}_decile')['model_x_human_agreement'].mean()
+    # Convert decile to range 0-1
+    p_label_grouped.index = p_label_grouped.index / 10
+    # Sort the DataFrame by confidence scores
+    label_df_sorted = label_df.sort_values(by=confidence_label)
+
+    p_label_moving_avg = label_df_sorted['model_x_human_agreement'].rolling(window=window_size).mean()
+
+    return label_df_sorted[confidence_label], p_label_moving_avg
+
+confidences = labels['probability']
+
+# Make dataframe of human label, model label, and confidence
+label_df = pd.DataFrame(data = {
+    'human_label': human_labels,
+    'decoder_label': predictions,
+    'confidence': confidences},
+    index = sorting_analyzer.sorting.get_unit_ids())
+
+# Calculate the proportion of agreed labels by confidence decile
+label_df['model_x_human_agreement'] = label_df['human_label'] == label_df['decoder_label']
+
+p_agreement_sorted, p_agreement_moving_avg = calculate_moving_avg(label_df, 'confidence', 3)
+
+# Plot the moving average of agreement
+plt.figure(figsize=(6, 6))
+plt.plot(p_agreement_sorted, p_agreement_moving_avg, label = 'Moving Average')
+plt.axhline(y=1/len(np.unique(predictions)), color='black', linestyle='--', label='Chance')
+plt.xlabel('Confidence'); #plt.xlim(0.5, 1)
+plt.ylabel('Proportion Agreement with Human Label'); plt.ylim(0, 1)
+plt.title('Agreement vs Confidence (Moving Average)')
+plt.legend(); plt.grid(True); plt.show()
+
+##############################################################################
+# In this case, you might decide to only trust labels which had confidence over above 0.86.
+#
+# A more realistic example: Neuropixels data
+# ------------------------------------------
+#
+# Above, we used a toy model trained on generated data. There are also models on HuggingFace
+# trained on real data.
+#
+# For example, the following classifier is trained on Neuropixels data from 11 mice recorded in
+# V1,SC and ALM: https://huggingface.co/AnoushkaJain3/curation_machine_learning_models
+#
+# Go to that page, and take a look at the ``Files``. The models are contained in the
+# `skops files <https://skops.readthedocs.io/en/stable/>`_ and there are *two* in this repo.
+# We can choose which to load in the ``load_model`` function as follows:
+#
+# .. code-block::
+#    import spikeinterface.curation as sc
+#    model, model_info = sc.load_model(
+#        repo_id = "AnoushkaJain3/curation_machine_learning_models",
+#        model_name= 'noise_neuron_model.skops',
+#    )
+#
+# If you run this locally you will receive an error:
```

Comment:
This section is a little long / technical. I would suggest highlighting the error and fix, and including details in a [dropdown](https://sphinx-design.readthedocs.io/en/latest/dropdowns.html).

e.g.
```
# If you run this locally you will receive an error:
#     UntrustedTypesFoundException: Untrusted types found in the file: ['sklearn.metrics._classification.balanced_accuracy_score', 'sklearn.metrics._scorer._Scorer', 'sklearn.model_selection._search_successive_halving.HalvingGridSearchCV', 'sklearn.model_selection._split.StratifiedKFold']
#
# This error is raised because it is possible to inject malicious code into the model format. If you unequivocally trust the model (e.g. if you have created it), you can bypass this security
# step by passing ``trust_model = True`` to the ``load_model`` function.
#
# In general, you should be cautious when downloading ``.skops`` files and ``.pkl`` files from repos,
# especially from unknown sources. For more information, see the below dropdown.
```

Reply:
done!